bitshares_pricefeed/pricefeed.py

```diff
@@ -2,14 +2,14 @@
 import numpy as num
 import time
 from pprint import pprint
-from math import fabs, sqrt
+from math import *
```

Comment:
That's not an ideal solution. Please define which modules you need when you import them directly.

Reply:
This is related to issue #37.

In this case only `sin` should be enough.
Unfortunately we don't know what user may need in their formula, that why I used `*`.
That's why I agree with @clockworkgr approach (and other pricefeed implementations), and use python modules for algorithmic assets instead of the configuration based approach. But I didn't want to change the approach in this pull request.